Bottom: e4d0836b1383f7c3802101b133dcb87ab52eeffe
Top:    1f2dac7787cba82282889fb425c8f9761db6c785
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-31 09:28:57 -0400

Refresh of device-trees

---

diff --git a/dt_config b/dt_config
index 825ff36..946862d 100644
--- a/dt_config
+++ b/dt_config
@@ -255,7 +255,7 @@ CONFIG_MMU=y
 # CONFIG_ARCH_VT8500 is not set
 # CONFIG_ARCH_ZYNQ is not set
 CONFIG_ARCH_LPC31XX=y
-# CONFIG_GPIO_PCA953X is not set
+CONFIG_GPIO_PCA953X=y
 
 #
 # System MMU
@@ -612,7 +612,8 @@ CONFIG_MTD_CFI_I2=y
 CONFIG_MTD_DATAFLASH=y
 # CONFIG_MTD_DATAFLASH_WRITE_VERIFY is not set
 # CONFIG_MTD_DATAFLASH_OTP is not set
-# CONFIG_MTD_M25P80 is not set
+CONFIG_MTD_M25P80=y
+CONFIG_M25PXX_USE_FAST_READ=y
 # CONFIG_MTD_SST25L is not set
 # CONFIG_MTD_SLRAM is not set
 # CONFIG_MTD_PHRAM is not set
@@ -1085,6 +1086,7 @@ CONFIG_GPIO_SYSFS=y
 #
 # CONFIG_GPIO_MAX7300 is not set
 # CONFIG_GPIO_MAX732X is not set
+CONFIG_GPIO_PCA953X_IRQ=y
 # CONFIG_GPIO_PCF857X is not set
 # CONFIG_GPIO_SX150X is not set
 # CONFIG_GPIO_ADP5588 is not set
@@ -1252,7 +1254,7 @@ CONFIG_SND_VERBOSE_PROCFS=y
 CONFIG_SND_SOC=y
 CONFIG_SND_LPC313X_SOC=y
 CONFIG_SND_LPC313X_SOC_I2S=y
-CONFIG_SND_EA3131_SOC_I2S_UDA1380=y
+CONFIG_SND_LPC313X_SOC_I2S_UDA1380=y
 CONFIG_SND_USE_EA3131=y
 CONFIG_SND_I2C1_CHANNEL_UDA1380=y
 CONFIG_SND_I2S_RX0_MASTER=y
